packages/server/utils/analytics/analytics.ts

```diff
@@ -141,6 +145,49 @@ class Analytics {
     })
   }
 
+  inviteEmailSent = (
+    userId: string,
+    teamId: string,
+    inviteeEmail: string,
+    isInviteeParabolUser: boolean,
+    inviteTo: 'meeting' | 'team',
+    success: boolean
+  ) => {
+    this.track(userId, 'Invite Email Sent', {
+      teamId,
+      inviteeEmail,
+      isInviteeParabolUser,
+      inviteTo,
+      success
+    })
+  }
+
+  inviteAccepted = (
```

Comment:
@tianrunhe I’m curious where we will be able to see if folks accepted via the link or the email?

Reply:
@ackernaut you can break down by `utm_medium`. [Example Amplitude chart](https://analytics.amplitude.com/parabol/chart/zta80j7?source=redirect%3A%20chart%20saved)

Reply:
I’m glad I asked! I didn’t see that, but it makes sense. Thank you